composer/algorithms/scale_schedule/scale_schedule.py

```diff
@@ -32,11 +31,11 @@ def __init__(self, ratio: float = 1.0):
             category=DeprecationWarning)
 
     def match(self, event: Event, state: State) -> bool:
-        """Run on Event.INIT.
+        """Run on :attr:`~composer.core.event.Event.INIT`.
```

Comment:
can you try :attr:`~Event.INIT`? That should work, and we want to the docstrings readable.

Reply:
Gotcha.

Reply:
@hanlint I just tried this, and it doesn't link to the class with the shortened syntax. :(